File: Src/Server/apcurium.MK.Web/admin/models/Tariff.js
(function(){

    var Tariff = TaxiHail.Tariff = Backbone.Model.extend({
        urlRoot: TaxiHail.parameters.apiRoot + '/admin/tariffs',
        defaults: function(){
            var today = new Date();
            return {
                name: '',
                minimumRate: 0,
                flatRate: 0,
                kilometricRate: 0,
                perMinuteRate: 0,
                marginOfError: 20,
                kilometerIncluded :0,
                startTime: TaxiHail.date.toISO8601(new Date(today.getYear(), today.getMonth(), today.getDate())),
                endTime: TaxiHail.date.toISO8601(new Date(today.getYear(), today.getMonth(), today.getDate() + 1)),
                daysOfTheWeek: 0,
                vehicleTypeId: null
            };
        }
    }, {
        type: {
            'default': 0,
            recurring: 1,
            day: 2,
            vehicleDefault: 3
        }
    });

    Tariff.prototype.defaults.type = Tariff.type.recurring;

}());

File: Src/Server/MK.Common/Entity/Tariff.cs
#region

using System;

#endregion

namespace apcurium.MK.Common.Entity
{
    public class Tariff
    {
        public Guid Id { get; set; }
        public Guid CompanyId { get; set; }
        public string Name { get; set; }
        public double MinimumRate { get; set; }
        public decimal FlatRate { get; set; }
        public double KilometricRate { get; set; }
        public double PerMinuteRate { get; set; }
        public double MarginOfError { get; set; }
        public double KilometerIncluded { get; set; }
        public int DaysOfTheWeek { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public int Type { get; set; }
        public int? VehicleTypeId { get; set; }
    }
}